frontend/src/screens/SingleProduct/index.js

```diff
@@ -56,7 +71,8 @@ const SingleProduct = ({ match: { params: { id } } }) => {
                                 <li className={style.list_item}><strong>{name}</strong></li>
                                 <li className={style.list_item}>{description}</li>
                                 <li className={style.list_item}>{countInStock > 0 ? 'In Stock' : 'Out of Stock'}</li>
-                                <li className={style.list_item}><Rating value={rating} numReviews={`${numReviews}`} /></li>
+                                <li className={style.list_item}>
```

Comment:
Any specific reason to use classnames in this fashion?
`style` is being repeated and de-structuring it would be tedious.

Reply:
This is modular css
This is used in this fashion only :)
Thanks